homeassistant/components/pioneer/media_player.py

```diff
@@ -55,6 +58,10 @@
         vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
         vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.socket_timeout,
         vol.Optional(CONF_SOURCES, default=DEFAULT_SOURCES): {cv.string: cv.string},
+        vol.Optional(
+            CONF_ALTERNATIVE_VOLUME_CONTROL, default=DEFAULT_ALTERNATIVE_VOLUME_CONTROL
+        ): cv.boolean,
+        vol.Optional(CONF_MAX_VOLUME, default=DEFAULT_MAX_VOLUME): cv.positive_int,
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

Reply:
I must say it seems like adding the configuration UI will change the scope of this PR quite a bit, but i will give it a shot. Thanks for the heads up

Reply:
Please make a separate PR for the config flow and config entries support. Thanks!

Reply:
Will do! This PR will be rebased once the other is accepted into `dev`